daal4py/sklearn/decomposition/_pca.py

```diff
@@ -193,8 +193,12 @@ def _fit(self, X):
                 self._fit_svd_solver = 'full'
 
         if self._fit_svd_solver == 'full':
-            logging.info("sklearn.decomposition.PCA.fit: " + get_patch_message("daal"))
-            return self._fit_full(X, n_components)
+            if X.shape[1]/X.shape[0] < 2:
```

Comment:
```suggestion
            if X.shape[1] / X.shape[0] < 2:
```
and fix CodeFactor comment